introduction_to_amazon_algorithms/object_detection_birds/object_detection_birds.ipynb

```diff
@@ -34,7 +34,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from utils import download
</pre><p>Why add a utils file with only one method and a few lines of code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3007/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>makes the notebook cleaner</p>